_includes/components/header--basic.html

```diff
@@ -0,0 +1,75 @@
+{% if header %}
+{% include banner.html %}
+<header class="usa-header usa-header-{{ header.type | default: 'extended' }}" role="banner">
+  {% if header %}
```

Comment:
This conditional seems unnecessary since it's already within an `{% if header %}`.